docs/platform/manager/an-overview-of-the-linode-cloud-manager/index.md

```diff
@@ -221,24 +197,19 @@ In Cloud Manager you can download a printable PDF of your invoice from your list
+### Password Management
 
-1.  Navigate to the **Account** page in the sidebar links menu, then click on the **Settings** tab.
+The Cloud Manager does not support forcing password expirations. Forcing password resets on a schedule is [bad practice from a security perspective](https://pages.nist.gov/800-63-FAQ/#q-b05). Current security research indicates that forced password changes do more harm than good. If you want to force password resets for users of your Linode account, we recommend using a password manager for this purpose.
 
-1.  Expand the panel labeled **Import Display Groups as Tags** and then click **Import Display Groups**:
+## Tags
 
-    ![Cloud Manager Import Display Groups](classic-to-cloud-cloud-import-display-groups.png "Cloud Manager Import Display Groups")
+Linode’s Cloud Manager and [API v4](https://developers.linode.com/api/v4) allow you to create tags to help organize and group your Linode resources. Tags can be applied to [Linodes](#linodes), [Block Storage Volumes](#volumes), [NodeBalancers](#nodebalancers), and [Domains](#domains-dns-manager). See the [Tags and Groups](/docs/quick-answers/linode-platform/tags-and-groups/) guide to learn how to create, apply, and search for tags.
 
-1.  A form will appear that lists your Display Groups and asks you to confirm the import action. To proceed, click the **Import Display Groups Now** button in this form.
+## Events and Activity Feeds
 
-    {{< note >}}
-Importing your Display Groups is a **one-time** operation. If you don't have any Display Groups configured in the Classic Manager this feature will not appear in the Cloud Manager.
-{{< /note >}}
+Tasks performed using the Linode Cloud Manager or other account specific tools like Linode’s [CLI](/docs/platform/api/linode-cli/) or [API](https://www.linode.com/products/api/) will be logged to an individual Linode’s activity feed, or on your account’s [Events Page](https://cloud.linode.com/events). The events and activity pages are user accessible logs, or histories of events taking place on your account. They contain details regarding the most notable events affecting your Linodes, like reboots, shutdowns, migrations, and more.
 
-### Password Management
-
-The Cloud Manager does not support forcing password expirations. Forcing password resets on a schedule is [bad practice from a security perspective](https://pages.nist.gov/800-63-FAQ/#q-b05). Current security research indicates that forced password changes do more harm than good. If you want to force password resets for users of your Linode account, we recommend using a password manager for this purpose.
+For more details, see the [Understanding the Cloud Manager Events and Activity Feeds](/docs/platform/manager/what-are-the-cloud-manager-events-and-activity-feeds/) guide.
 
 ## My Profile
 
```

Comment:
After Resetting your Account password it says "managing trusted devices." This I think is typo, which should have been caught in a previous review.

Reply:
done